src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
+const array = [];
+
+const populations = document.querySelectorAll('.population');
+
+populations.forEach((population) => {
+  const num = Number(population.textContent.replace(/,/g, ''));
```

Comment:
Ensure that the text content of the population elements is always a valid number after removing commas. If there's a chance of invalid data, consider adding error handling.